csharp/test/Ice/inactivityTimeout/AllTests.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) ZeroC, Inc.
+
+#nullable enable
+
+namespace Ice.inactivityTimeout;
+
+internal class AllTests : global::Test.AllTests
+{
+    internal static async Task allTests(global::Test.TestHelper helper)
+    {
+        Communicator communicator = helper.communicator();
+        string proxyString = $"test: {helper.getTestEndpoint()}";
+        Test.TestIntfPrx p = Test.TestIntfPrxHelper.createProxy(communicator, proxyString);
+
+        string proxyString3s = $"test: {helper.getTestEndpoint(1)}";
+
+        await testClientInactivityTimeout(p, helper.getWriter());
+        await testServerInactivityTimeout(proxyString3s, communicator.getProperties(), helper.getWriter());
+        await testWithOutstandingRequest(p, oneway: false, helper.getWriter());
+        await testWithOutstandingRequest(p, oneway: true, helper.getWriter());
+
+        await p.shutdownAsync();
+    }
+    private static async Task testClientInactivityTimeout(Test.TestIntfPrx p, TextWriter output)
+    {
+        output.Write("testing that the client side inactivity timeout shuts down the connection... ");
+        output.Flush();
+
+        await p.ice_pingAsync();
+        Connection connection = p.ice_getConnection();
+        test(connection is not null);
+
+        // The inactivity timeout is 3s on the client side and 5s on the server side. 4 seconds tests the client side.
+        await Task.Delay(4000);
+        await p.ice_pingAsync();
+        Connection connection2 = p.ice_getConnection();
+        test(connection2 != connection);
+        output.WriteLine("ok");
+    }
+
+    private static async Task testServerInactivityTimeout(string proxyString, Properties properties, TextWriter output)
+    {
+        output.Write("testing that the server side inactivity timeout shuts down the connection... ");
+        output.Flush();
+
+        // Create a new communicator with the desired properties.
+        properties = properties.Clone();
+        properties.setProperty("Ice.Connection.InactivityTimeout", "3");
```

Comment:
Shouldn't we set it to `5` as mentioned in the comment below?

```suggestion
        properties.setProperty("Ice.Connection.InactivityTimeout", "5");
```

Reply:
Fixed!